src/m3/test/self-definition.ts

```diff
@@ -35,14 +40,14 @@ namespaceProvider.havingFeatures(namespaceProvider_namespaceQualifier)
 
 const namespacedEntity = new Concept(lioncore, "NamespacedEntity", true)
 
-const namespacedEntity_simpleName = new Property(namespacedEntity, "simpleName", Multiplicity.Single)
+const namespacedEntity_simpleName = new Property(namespacedEntity, "simpleName")
     .ofType(stringDatatype)
 
-const namespacedEntity_qualifiedName = new Property(namespacedEntity, "qualifiedName", Multiplicity.Single)
+const namespacedEntity_qualifiedName = new Property(namespacedEntity, "qualifiedName")
```

Comment:
Do we have derived features? Here I cannot see them: https://github.com/LIonWeb-org/organization/blob/main/lioncore/metametamodel.svg

Reply:
Right, they've been removed there. But that makes this diagram not definable in M3/lioncore itself. I'll have to think about that a bit.

Reply:
Maybe we can leave them for now and mark them as "Experimental"?

Reply:
I'll see whether I do the marking with an annotation or something otherwise (JS doesn't really have annotations in the way that Java/the JVM has).